base_fontawesome/templates/assets.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+Copyright 2017 Simone Orsi
+License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+<odoo>
+
+	<template id="assets_backend" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+			<link rel="stylesheet" type="text/css" href="/base_fontawesome/static/lib/fontawesome-5.3.1/css/all.css" />
```

Comment:
can we get rid of unused resources? :)

Reply:
If I replace the elements it is giving problems, related to the usage of FontAwesome font directly on some code by odoo. I will try to review if it is possible to fix the problem.